### PR TITLE
1564 fix failed travis tests

### PR DESCRIPTION
**Pull request by dexguitar**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818033&bid=16020567)

File: jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java [hunk 1]
```diff
@@ -9,11 +9,17 @@
...
 
 public class HomePageEx extends PageBase {
-    public HomePageEx() { super("/index.html", "Home");}
-    @FindBy(css="form #name") public WebElement userName;
+    public HomePageEx() {
+        super("/index.html", "Home");
+    }
+
+    @FindBy(css = "#name")
+    public WebElement userName;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818033)

File: jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java [hunk 2]
```diff
@@ -9,11 +9,17 @@
...
     public WebElement password = $("form #password");
 
-    @UI("form #name") public UIElement userNameUI;
+    @UI("#name")
+    public UIElement userNameUI;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818033)

File: jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java [hunk 3]
```diff
@@ -9,11 +9,17 @@
...
     public UIElement passwordUI = $("form #password");
-    @UI("#login-button") public UIElement enterUI;
+    @UI("#login-button")
+    public UIElement enterUI;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818033)

**Comment by mf-code** [comment]
[4482] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4482] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4482] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4482] - Allure report on Netlify: https://5e1ee1829ae4ef6de44d2eed--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4484] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4484] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4484] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4484] - Allure report on Netlify: https://5e1ee8a8e624aef924efb4cf--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4492] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4492] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4492] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4492] - Allure report on Netlify: https://5e1f542b15dc0aa8a451b167--musing-swartz-6fc060.netlify.com

**Comment by dexguitar** [comment]
Closed for time being...

**Comment by mf-code** [comment]
[4545] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4545] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4545] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4545] - Allure report on Netlify: https://5e25c4ac5b5e234281d34256--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4552] - https://transfer.sh/f6Jeq/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4552] - https://transfer.sh/bdkZs/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4552] - https://transfer.sh/U0qOm/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4552] - Allure report on Netlify: https://5e271f8909b9ffcba065b0bf--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4564] - https://transfer.sh/4DFHv/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4564] - https://transfer.sh/ouJj5/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4564] - https://transfer.sh/d3IT1/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4564] - Allure report on Netlify: https://5e295e9a47c8937f68186c8c--musing-swartz-6fc060.netlify.com

**Pull request closed without merging**